lualib/lua_cfg_transform.lua

```diff
@@ -396,7 +396,7 @@ return function(cfg)
 
     local actions_set = lua_util.list_to_hash(actions_defs)
 
-    -- Now check actions section for garbadge
+    -- Now check actions section for garbage
```

Comment:
I was wondering if this was a term of art, but I didn't find anything obvious for it...

Reply:
This is my favourite spelling mistake. I don't know why :)